app/views/layouts/application.html.erb

```diff
@@ -9,11 +9,44 @@
   </head>
 
   <body>
+    <% if session[:user_id] != nil %>
+      <% user = User.find(session[:user_id]) %>
+    <% end %>
     <nav class = "topnav">
-      <%= link_to "All Merchants", "/merchants"%>
-      <%= link_to "All Items", "/items"%>
-      <%= link_to "Cart: #{cart.total_items}", "/cart" %>
-      <%= link_to "Register", "/register" %>
+      <% if user == nil %>
```

Comment:
Could this stuff be above the if statements (and removed from being repeated in each if statement)?
`<%= link_to "Home", "/"%>
      <%= link_to "All Merchants", "/merchants"%>
      <%= link_to "All Items", "/items"%>`

Reply:
I like that idea!  Make it accessible so there's less repetition.

Reply:
Great idea Kiera!